### PR TITLE
Updates Make Target in Contrib Doc

### PR DESCRIPTION
**Pull request by danehans**
Updates the contrib guide to use the `check` target.

xref: https://github.com/projectcontour/contour-operator/issues/9

/assign @stevesloka @skriss @jpeach 
/cc @Miciah

### CODE REVIEW

**Review by skriss — APPROVED** [review]
LGTM

File: Makefile [hunk 1]
```diff
@@ -64,12 +64,8 @@ endif
 
 all: manager
 
-# Run tests & validate against linters
-.PHONY: check
-check: test lint-golint
-
 # Run tests
-test: generate fmt vet manifests
+test: generate fmt vet lint-golint manifests
```
Comment (stevesloka) [inline Makefile:68]: FYI, this change is going to run golint-ci twice now, once in `make test` as well as once for the GH action (https://github.com/projectcontour/contour-operator/blob/main/.github/workflows/prbuild.yaml#L25-L33).

Comment (danehans) [reply Makefile]: @stevesloka thanks for catching this. Commit `7db0230` keeps the existing make targets and simply updates the contrib guide to use the appropriate target.

Comment (stevesloka) [reply Makefile]: If you prefer to use `make test` we can swap things around. I'll merge this and can follow up with another PR if desired.

**Review by stevesloka — APPROVED** [review]
/lgtm

**Pull request merged**